packages/server/test/e2e/1_typescript_plugins_spec.ts

```diff
@@ -0,0 +1,19 @@
+import e2e from '../support/helpers/e2e'
+import Fixtures from '../support/helpers/fixtures'
+
+describe('e2e typescript in plugins file', function () {
+  e2e.setup()
+
+  it('handles tsconfig with module other than commonjs', function () {
+    return e2e.exec(this, {
+      project: Fixtures.projectPath('ts-proj-with-module-esnext'),
+    })
+  })
+
+  // https://github.com/cypress-io/cypress/issues/7575
+  it('allows esModuleInterop to be overridden', function () {
+    return e2e.exec(this, {
+      project: Fixtures.projectPath('ts-proj-esmoduleinterop-false'),
+    })
+  })
+})
```

Comment:
shouldn't the tests around this be:
- one test with `esmoduleinterop: true` (testing that users can override)
- one test with no value set (testing that default is unset, or false)